### DISCUSSION TITLE
Can you please help me build a smart_contract suitable for the above project?

### DISCUSSION THREAD

**Discussion by dungnd123** [DevelopersLeague/EtherDocs#26] [General]
Hello, have a nice day
How are you? I really need help from you.
I have a small assignment about EtherDocs, I saw your project on github page. I'm learning to do my project, but I'm having difficulty with smart_contract.
Can you please help me build a smart_contract suitable for the above project?
Hope you reply soon. Thank you very much.

**Comment by ShyrenMore** [comment]
Hi @dungnd123 hope I am not too late, we have intentionally hidden smart contract folder in the repo, contact us on [Telegram](https://telegram.me/ShyrenMore) to gain access
